synapse/python_dependencies.py

```diff
@@ -73,15 +73,17 @@
     "resources.consent": ["Jinja2>=2.9"],
 
     "saml2": ["pysaml2>=4.5.0"],
+    "systemd": ["systemd-python>=231"],
     "url_preview": ["lxml>=3.5.0"],
     "test": ["mock>=2.0"],
 }
 
 
 def list_requirements():
     deps = set(REQUIREMENTS)
-    for opt in CONDITIONAL_REQUIREMENTS.values():
-        deps = set(opt) | deps
+    for name, opts in CONDITIONAL_REQUIREMENTS.items():
+        if name not in ["systemd"]:
```

Comment:
I'm not sure what the point of including `systemd` in CONDITIONAL_REQUIREMENTS in the first place is?

Reply:
oh wait, now I do. But I don't think changing this will change the behaviour for `[all]`, which seems to be done in `setup.py`.

Reply:
adfghdfh yes of course it won't :/